### PR TITLE
Fix org redirection

### PR DESCRIPTION
**Pull request by ivanmalagon**
With the latest static deploy, org users using a subdomain URL were not redirected to the right place.

This PR checks if the user has an organization and if the username equals the subdomain to perform the redirection client side.

After seeing the code, I don't think domainless URLS for OnPrem are not affected since they don't use a subdomain. Correct if I'm wrong @javitonino

### CODE REVIEW

**Comment by javitonino** [comment]
domainless URLs don't have any redirection, so I'm guessing they are OK. But they should be tested during acceptance.

A question about the behaviour/code (not really CR). Do we haev static public pages? The implementation there could be different (since the visited user does not match the logged-in user, you cannot use `userData` there). I _think_ that for dashboard it is fine, but make sure to test the case where you are logged-in as a user (or logged-out) but visit the page for a different org/user. As I said, I'm pretty sure this is fine as long as we don't have public page statics, but as soon as we add those, things are gonna get complicated :)

**Comment by ivanmalagon** [comment]
Static pages are `profile`, `account` and `dashboard`. All of them are only private.

File: lib/assets/javascripts/dashboard/account.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const ConfigModel = require('dashboard/data/config-model');
 const UserModel = require('dashboard/data/user-model');
 const OrganizationModel = require('dashboard/data/organization-model');
 const AssetsVersionHelper = require('dashboard/helpers/assets-version');
+const redirectOrgUsers = require('dashboard/helpers/redirector').redirectOrgUsers;
```
Comment (rjimenezda) [inline lib/assets/javascripts/dashboard/account.js]: You can write this as

const { redirectOrgUsers } = require('dashboard/helpers/redirector')

**Comment by ivanmalagon** [comment]
Redirection to login to be made in a new PR.

**Pull request merged**